tests/src/data/use-cases/db-forget-password.spec.ts

```diff
@@ -86,8 +89,8 @@ describe('DbForgetPassword UseCase', () => {
   })
 
   it('Should return false if LoadGuardianByEmail returns undefined', async () => {
-    const { sut, loadGuardianByEmailStub } = makeSut()
-    jest.spyOn(loadGuardianByEmailStub, 'loadByEmail').mockReturnValueOnce(Promise.resolve(undefined))
+    const { sut, guardianRepositoryStub } = makeSut()
+    jest.spyOn(guardianRepositoryStub, 'loadByEmail').mockReturnValueOnce(Promise.resolve(undefined))
```

Comment:
Here I could resolve this Promise with just undefined.

Reply:
I agree, at this point I focused in name refactor.